server/playground/index.js

```diff
@@ -0,0 +1,33 @@
+const buildSchema = require("@aerogear/data-sync-gql-core");
+const { makeExecutableSchema } = require("graphql-tools");
+const { ApolloServer } = require("apollo-server-express");
+const { log } = require("../logger");
+const models = require("../models");
+
+async function connectDataSources(dataSources) {
```

Comment:
In my mind this should be responsibility of the core module (however I do not have strong opinion about it). See my previous proposition in comment for end user api.